sklearn/tests/test_pipeline.py

```diff
@@ -125,6 +133,17 @@ def score(self, X, y=None, sample_weight=None):
         return np.sum(X)
 
 
+class DummyTransf(Transf):
+    """Transformer which store the column means"""
+
+    def fit(self, X, y):
+        self.means_ = np.mean(X, axis=0)
+        # Store a timestamp such that we know
+        # that we have a cache object
+        self.timestamp = time.time()
```

Comment:
Please rename `self.timestamp` to `self.timestamp_` to follow the naming convention for fitted parameters.